lms/djangoapps/instructor_task/api.py

```diff
@@ -25,6 +25,8 @@
 
 from instructor_task.api_helper import (check_arguments_for_rescoring,
                                         encode_problem_and_student_input,
+                                        encode_entrance_exam_and_student_input,
+                                        check_entrance_exam_problems_for_rescoring,
                                         submit_task)
```

Comment:
nitpick: similar to other whitespacing feedback, typically we have that '(' at the top line, then all imported modules indented one from the left. Then the last ')' would be fully left aligned.